### PR TITLE
BIP68: update specification to assume MTP

### PR DESCRIPTION
**Pull request by btcdrak**
BIP68 and BIP113 will be deployed together which simplifies the specification slightly

### CODE REVIEW

File: bip-0068.mediawiki [hunk 1]
```diff
@@ -30,7 +30,7 @@ If bit (1 << 31) of the sequence number is not set, then the sequence number is
 
 The sequence number encoding is interpreted as follows: 
 
-Bit (1 << 22) determines if the relative lock-time is time-based or block based: If the bit is set, the relative lock-time specifies a timespan in units of 512 seconds granularity. The timespan starts from the median-time-past (MTP) of the output’s previous block, and ends either at the MTP of the previous block or at the nTime of the transaction’s block (depending on the enforcement status of BIP113). If the bit is not set, the relative lock-time specifies a number of blocks.
+Bit (1 << 22) determines if the relative lock-time is time-based or block based: If the bit is set, the relative lock-time specifies a timespan in units of 512 seconds granularity. The timespan starts from the median-time-past (MTP) of the output’s previous block, and ends either at the MTP of the previous block or at the nTime of the transaction’s block as defined by BIP113. If the bit is not set, the relative lock-time specifies a number of blocks.
```
Comment (morcos) [inline bip-0068.mediawiki]: I would just say this: ", and ends at the MTP of the previous block."   and get rid of the "or  ... BIP113"

File: bip-0068.mediawiki [hunk 2]
```diff
@@ -43,7 +43,7 @@ For time based relative lock-time, 512 second granularity was chosen because bit
 When the relative lock-time is time-based, it is interpreted as a minimum block-time constraint over the input's age. A relative time-based lock-time of zero indicates an input which can be included in any block. More generally, a relative time-based lock-time n can be included into any block produced 512 * n seconds after the mining date of the output it is spending, or any block thereafter.
 The mining date of the output is equals to the median-time-past of the previous block which mined it.
 
-The block produced time is either equals to median time past of its parent or to its nTime field, depending on the state of BIP113 median-time-past.
+The block produced time is equal to the median-time-past of its parent according to BIP113.
```
Comment (morcos) [inline bip-0068.mediawiki]: "The block produced time is equal to the median-time-past of its pervious block."

File: bip-0068.mediawiki [hunk 3]
```diff
@@ -191,7 +191,7 @@ This BIP was edited by BtcDrak, Nicolas Dorier and kinoshitajona.
 
 This BIP is to be deployed by either version-bits BIP9 or by isSuperMajority(). Exact details TDB.
 
-It is recommended to deploy BIP112 and BIP113 at the same time as this BIP.
+BIP68 MUST be deployed together with BIP113. It is also recommended to deploy BIP112 at the same time.
```
Comment (morcos) [inline bip-0068.mediawiki]: not necessary, but fine.

**Comment by afk11** [comment]
ACK

**Comment by petertodd** [comment]
ACK

**Comment by NicolasDorier** [comment]
Well, too bad I did not see that before, it broke completely #6312, which was well reviewed and assumed same MTP policy for both nLockTime and sequence lock time.

**Comment by btcdrak** [comment]
@NicolasDorier This was agreed at the IRC meeting. I am leaning for 7184 to replace 6312 in any case.

**Pull request merged**